.eslintrc.js

```diff
@@ -59,49 +65,54 @@ module.exports = {
         'prefer-const': ['error', {}],
         'react/jsx-closing-bracket-location': ['error', {
             'nonEmpty': 'after-props',
-            'selfClosing': 'line-aligned',
+            'selfClosing': 'line-aligned'
         }],
         'react/jsx-closing-tag-location': 'error',
         'react/jsx-curly-brace-presence': 'error',
         'react/jsx-curly-spacing': ['error', {
             'children': true,
             'spacing': {
-                'objectLiterals': 'never',
+                'objectLiterals': 'never'
             },
-            'when': 'always',
+            'when': 'always'
         }],
         'react/jsx-equals-spacing': 'error',
         'react/jsx-handler-names': ['error', {
             'eventHandlerPrefix': 'on',
-            'eventHandlerPropPrefix': 'on',
+            'eventHandlerPropPrefix': 'on'
         }],
         'react/jsx-indent': 'error',
         'react/jsx-indent-props': 'error',
         'react/jsx-no-target-blank': 'error',
         'react/jsx-props-no-multi-spaces': 'error',
+        'react/jsx-sort-props': ['error', {
+          'ignoreCase': true,
+          'reservedFirst': true
+        }],
         'react/jsx-tag-spacing': ['error', {
             'beforeClosing': 'never',
-            'beforeSelfClosing': 'allow',
+            'beforeSelfClosing': 'allow'
         }],
         'react/jsx-wrap-multilines': 'error',
         'react/no-deprecated': 'error',
         'react/no-typos': 'error',
         'react/no-unknown-property': 'error',
         'react/no-unsafe': ['error', {
-            'checkAliases': true,
+            'checkAliases': true
         }],
         'react/no-unused-prop-types': 'error',
         'react/prefer-stateless-function': 'error',
         'react/prop-types': 'off',
         'react/react-in-jsx-scope': 'off',
-        'semi': ['error', 'always'],
+        'semi': 'off',
```

Comment:
Why was this disabled?

Reply:
Can't find were I read about this but I think it was to avoid conflicts with `@typescript-eslint/member-delimiter-style`.

Reply:
But does disabling this not also mean that semicolons are not required anywhere? I mean, every line should end with a semicolon, not just `type`/`interface` delimiters.

Reply:
I tried removing semicolons at the end of different lines and it works, generating an error. If I change this back to `always` the same error is generated twice, so they seem to do the same thing in this regard.

Reply:
Sorry, had forgotten to remove the code on line 43 which made my tests invalid. I'll set `semi` to `['error', 'always']`, it seems to be working together with `@typescript-eslint/member-delimiter-style` not rendering double errors.